ui/src/app/applications/components/application-details/application-details.tsx

```diff
@@ -197,43 +211,45 @@ export class ApplicationDetails extends React.Component<RouteComponentProps<{nam
-                                                                services.viewPreferences.updatePreferences({appDetails: {...pref, view: 'pods'}});
+                                                                this.appContext.apis.navigation.goto('.', {view: AppsDetailsViewKey.Pods});
+                                                                services.viewPreferences.updatePreferences({appDetails: {...pref, view: AppsDetailsViewKey.Pods}});
                                                             }}
                                                         />
                                                         <i
-                                                            className={classNames('fa fa-network-wired', {selected: pref.view === 'network'})}
+                                                            className={classNames('fa fa-network-wired', {selected: pref.view === AppsDetailsViewKey.Network})}
                                                             title='Network'
                                                             onClick={() => {
-                                                                this.appContext.apis.navigation.goto('.', {view: 'network'}, {replace: true});
-                                                                services.viewPreferences.updatePreferences({appDetails: {...pref, view: 'network'}});
+                                                                this.appContext.apis.navigation.goto('.', {view: AppsDetailsViewKey.Network});
+                                                                services.viewPreferences.updatePreferences({appDetails: {...pref, view: AppsDetailsViewKey.Network}});
                                                             }}
                                                         />
                                                         <i
-                                                            className={classNames('fa fa-th-list', {selected: pref.view === 'list'})}
+                                                            className={classNames('fa fa-th-list', {selected: pref.view === AppsDetailsViewKey.List})}
                                                             title='List'
                                                             onClick={() => {
-                                                                this.appContext.apis.navigation.goto('.', {view: 'list'}, {replace: true});
-                                                                services.viewPreferences.updatePreferences({appDetails: {...pref, view: 'list'}});
+                                                                this.appContext.apis.navigation.goto('.', {view: AppsDetailsViewKey.List});
+                                                                services.viewPreferences.updatePreferences({appDetails: {...pref, view: AppsDetailsViewKey.List}});
                                                             }}
```

Comment:
Maybe we can do a destructuring assignment for consts in `AppsDetailsViewKey` so that we don't have to repeat that many times?

`const {Tree, Pods, Network, List} = AppsDetailsViewKey;`

Reply:
Done